libs/expo/shared/ui-components/src/lib/Textarea/Textarea.tsx

```diff
@@ -121,6 +123,7 @@ export function Textarea(props: ITextareaProps) {
                 fontFamily: 'Poppins-Regular',
                 fontSize: FontSizes.md.fontSize,
                 lineHeight: FontSizes.md.lineHeight,
+                textAlignVertical: textAlignVertical,
```

Comment:
It looks like `textAlignVertical` is also accepted as a style? I thought it'd be used as a `TextInput` prop (not a style).
Is it the same in both cases?

Reply:
Hey Tom, thanks for questioning this. It works on both after testing but I moved to be its own property as is expected based on how its used from the component itself.